lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -82,6 +83,11 @@ def self.available_options
                                        optional: true,
                                        default_value: false,
                                        is_string: false),
+          FastlaneCore::ConfigItem.new(key: :githubtoken,
```

Comment:
💄 Nit: Ruby naming conventions use snake_case for parameters and variable names.
```suggestion
          FastlaneCore::ConfigItem.new(key: :github_token,
```

Reply:
Thanks, I'll address that!